lib/elixir/test/elixir/process_test.exs

```diff
@@ -15,9 +15,13 @@ defmodule ProcessTest do
   end
 
   test "info/2" do
-    pid = spawn fn -> end
+    pid = spawn fn -> :timer.sleep(1000) end
+    assert Process.info(pid, :status) == {:status, :runnable}
```

Comment:
I think there might be a little chance of race condition in which `:status` will be returned as `: running `.